app/renderer/components/preferences/payment/enabledContent.js

```diff
@@ -290,15 +295,15 @@ const styles = StyleSheet.create({
     marginBottom: 0
   },
 
-  iconLink: {
+  balance__iconLink: {
     textDecoration: 'none',
 
     ':hover': {
       textDecoration: 'none'
     }
   },
 
-  iconText: {
+  balance__iconLink__iconText: {
     color: globalStyles.color.mediumGray,
     margin: '0 0 0 5px',
 
```

Comment:
below we have `fontSize: `${globalStyles.fontSize.settingItemSubtext} !important`` is that still needed?